tests/ha/test_ha_pod_restart.py

```diff
@@ -2959,13 +2960,23 @@ def test_continuous_reads_during_pod_restart_ros(self):
         assert_utils.assert_true(resp[0], resp)
         LOGGER.info("Step 8: Cluster is in good state. All the services are up and running")
         event.clear()
+        LOGGER.debug("Event is cleared")
         thread.join()
-
+        LOGGER.debug("Thread has joined.")
```

Comment:
I think, these debug logs can be removed. Lines 2963 and 2965.

Reply:
> I think, these debug logs can be removed. Lines 2963 and 2965.

Observing thread getting stuck. Needed to have this line to make sure where the test getting stuck.